### ISSUE TITLE
Getting tf.nn.max_pool error in model.py

### ISSUE THREAD

**Issue by vliu15** [allenai/bilm-tf#102]
I have adjusted the max chars per token parameter in the `options.json` file to 16, and the `tf.nn.max_pool` on line 378 in `bilm/model.py` does not reduce the second axis dimension to 1, which causes an error a few lines later when `tf.squeeze` is performed on `axis=2`. Any insight?
